frontend/assets/css/colors.scss

```diff
@@ -23,12 +23,15 @@
   --blue: #2e82ae;
   --light-blue: #66bce9;
   --melllow-blue: #7DB5EC;
+  --sky-blue:#CDE3EE;
```

Comment:
fyi: spoke with @d-dog about the `hex casing`. In `figma` there is no option to change to to [lower case](https://forum.figma.com/t/option-to-show-color-with-lowercase-hex-instead-of-uppercase/2547/7)

we could:

- leave as is (mixed)
- add [stylelint](https://stylelint.io/)
- change it manually ( :-1:  )

Reply:
from my site i am gonna leave it as it is. We should circle to this topic back :)

Reply:
:ok:  Would be glad if you bring that up